lib/ansible/modules/system/systemd.py

```diff
@@ -300,6 +300,10 @@ def main():
         if rc != 0:
             module.fail_json(msg='failure %d during daemon-reload: %s' % (rc, err))
 
+        # exit if daemon-reload only
+        if module.params['state'] is None and module.params['enabled'] is None and module.params['masked'] is None:
+            module.exit_json(**result)
+
```

Comment:
yes, this is equivalent to the 3 ifs that follow, there is no need for this duplication